tiledb/sm/c_api/tiledb.h

```diff
@@ -5509,6 +5509,18 @@ TILEDB_DEPRECATED_EXPORT int32_t tiledb_array_consolidate_metadata_with_key(
     uint32_t key_length,
     tiledb_config_t* config);
 
+/**
+ * Gets then number of MBRs in an open array. The array must
+ * be opened in READ mode, otherwise the function will error out.
+ *
+ * @param ctx The TileDB context.
+ * @param array An array opened in READ mode.
+ * @param num The number of MBRs to be retrieved.
+ * @return `TILEDB_OK` for success and `TILEDB_ERR` for error.
+ */
+TILEDB_EXPORT int32_t tiledb_array_get_mbr_num(
```

Comment:
Is the number of MBRs in a fragment always equal to the number of physical tiles? I thought it might be equal to or less since there might be tiles that contain only empty cells.

Reply:
Yes, but probably that particular function is just a remnant of old code? It's not implemented and it is specific to an `array` instead of `fragment_info`. An array may have multiple fragments and each fragment a different number of MBRs. I guess I am confused about what this function is doing.

Reply:
Oh I see now! Yes this is old code from when I originally thinking to implement this under `array.h` instead of `fragment_info.h`. I didn't read this closely enough and thought you were asking about `tiledb_fragment_info_get_mbr_num`.